deploy/openshift/templates/che-server-template.yaml

```diff
@@ -0,0 +1,279 @@
+            value: 'true'
+          - name: CHE_INFRA_OPENSHIFT_TLS__ENABLED
+            value: "${TLS}"
+          - name: CHE_INFRA_KUBERNETES_TRUST__CERTS
+            value: "${TLS}"
+          - name: CHE_INFRA_KUBERNETES_USERNAME
+            value: "${OPENSHIFT_USERNAME}"
+          - name: CHE_LOCAL_CONF_DIR
+            value: "/etc/conf"
+          - name: CHE_LOGS_DIR
+            value: "/data/logs"
+          - name: CHE_LOG_LEVEL
+            value: INFO
+          - name: CHE_MULTIUSER
+            value: "${CHE_MULTIUSER}"
+          - name: CHE_KEYCLOAK_AUTH__SERVER__URL
+            value: "${CHE_KEYCLOAK_AUTH__SERVER__URL}"
+          - name: CHE_JDBC_URL
+            value: "${CHE_JDBC_URL}"
+          - name: CHE_OAUTH_GITHUB_CLIENTID
+          - name: CHE_OAUTH_GITHUB_CLIENTSECRET
+          - name: CHE_PREDEFINED_STACKS_RELOAD__ON__START
+            value: 'true'
+          - name: JAVA_OPTS
+            value: "-XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+              -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+              -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true
+              -Xms20m "
+          - name: CHE_WORKSPACE_AUTO_START
+            value: 'false'
+          - name: CHE_INFRA_KUBERNETES_PVC_QUANTITY
+            value: ${CHE_INFRA_KUBERNETES_PVC_QUANTITY}
+          - name: PROTOCOL
+            value: "${PROTOCOL}"
+          - name: ROUTING_SUFFIX
+            value: "${ROUTING_SUFFIX}"
+          image: ${IMAGE_CHE}:${CHE_VERSION}
+          imagePullPolicy: "${PULL_POLICY}"
+          livenessProbe:
+            httpGet:
+              path: /api/system/state
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 120
+            timeoutSeconds: 10
+          name: che
+          ports:
+          - containerPort: 8080
+            name: http
+          - containerPort: 8000
+            name: http-debug
+          readinessProbe:
+            httpGet:
+              path: /api/system/state
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 15
+            timeoutSeconds: 60
+          name: che
+          ports:
+          - containerPort: 8080
+            name: http
+            protocol: TCP
+          - containerPort: 8000
+            name: http-debug
+            protocol: TCP
+          resources:
+            limits:
+              memory: 1Gi
+            requests:
+              memory: 256Mi
+          terminationMessagePath: "/dev/termination-log"
+          terminationMessagePolicy: File
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        serviceAccount: che
+        serviceAccountName: che
+        terminationGracePeriodSeconds: 30
+    test: false
+    triggers:
+    - type: ConfigChange
+  status:
```

Comment:
Are you sure that it makes sense to specify status here?

Reply:
Removed